### PR TITLE
feat: add order DSL

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

**Review by tangjeff0 — COMMENTED** [review]
:fire: :fire: How about in the case of batch operations? e.g. remove 5 items.

**Comment by filipesilva** [comment]
@tangjeff0 can always do a reduce over the individual operations. Less performant though.

File: src/cljc/athens/common_events/resolver/order.cljc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+(ns athens.common-events.resolver.order
+  (:refer-clojure :exclude [remove])
+  (:require
+    [clojure.core :as c]))
+
+
+(defn remove
```
Comment (filipesilva) [inline src/cljc/athens/common_events/resolver/order.cljc:7]: @neotyk mentioned he'd like better docstrings here.

File: src/cljc/athens/common_db.cljc [hunk 2]
```diff
@@ -74,6 +74,17 @@
...
 
+(defn get-sidebar-titles
+  [db]
+  (->> (d/q '[:find [(pull ?e [*]) ...]
+              :where
+              [?e :page/sidebar _]]
+            db)
+       (sort-by :page/sidebar)
+       (map :node/title)
+       vec))
```
[the remark this answers is not in the document]
Comment (tangjeff0) [reply src/cljc/athens/common_db.cljc:85]: reuse `get-sidebar-elements`?

**Review by tangjeff0 — APPROVED** [review]
this PR is BUSSIN.

After merging this into [undo shortcuts](https://github.com/athensresearch/athens/pull/1972), this does appear to work. Would still appreciate a higher bandwidth walkthrough.

File: src/cljc/athens/common_db.cljc (hunk 2 above)
[the remark this answers is not in the document]
Comment (filipesilva) [reply src/cljc/athens/common_db.cljc]: done

File: src/cljc/athens/common_db.cljc [hunk 3]
```diff
@@ -221,6 +174,18 @@
...
+(defn get-children-uids
+  "Fetches page or block sorted children uids based on eid lookup."
+  [db eid]
+  (->> (d/pull db '[{:block/children [:block/uid
+                                      :block/order]}]
+               eid)
+       :block/children
+       (sort-by :block/order)
+       (map :block/uid)
+       vec))
```
[the remark this answers is not in the document]
Comment (filipesilva) [reply src/cljc/athens/common_db.cljc]: done

**Pull request merged**